src/view/components/molecules/NftModal.tsx

```diff
@@ -13,36 +15,74 @@ interface Props {
 }
 
 const NftModal: React.FC<Props> = (props): JSX.Element => {
+    const modalBody = {
+        backgroundColor: "#1b202c",
+        height: "80vh",
+        width: "70vw",
+        display: "inline-block",
+        borderRadius: "30px",
+        overflow: "hidden"
+    }
+
+    const nftData = {
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        textAlign: "start",
+        color: "#feda03",
+        padding: "2% 5%"
+    }
+
+    const popup = {
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center"
+    }
+
     return (
         <Modal
             open={props.open}
             onClose={props.closer}
             aria-labelledby="modal-modal-title"
             aria-describedby="modal-modal-description"
-            className="popup"
+            sx={popup}
         >
-            <Box className="modal-body">
-                <img className="heroImg" src={props.img} alt={props.title} />
-                <div className="nftData">
-                    <h1
-                        style={{
-                            fontWeight: "bold",
-                            textAlign: "center"
-                        }}
+            <Box sx={modalBody}>
+                <img
+                    className={styles.heroImg}
+                    src={props.img}
+                    alt={props.title}
+                />
+                <Box sx={nftData}>
+                    <Typography
+                        variant="h3"
+                        sx={{ fontWeight: "bold" }}
+                        gutterBottom
                     >
                         {props.title} #{props?.id}
-                    </h1>
-                    <div className="description">
-                        <h3>Description</h3>
-                        <p>{props.description}</p>
-                    </div>
-                    <a
-                        href={`https://testnets.opensea.io/assets/goerli/${props.address}/${props.id}`}
-                        target="_blank"
-                    >
-                        <button className="buy">Purchase on Opensea</button>
-                    </a>
-                </div>
+                    </Typography>
+                    <Box>
+                        <Typography variant="h5">Description</Typography>
+                        <Typography
+                            variant="body1"
+                            sx={{ color: "rgb(255, 246, 194)" }}
+                            gutterBottom
+                        >
+                            {props.description.length
+                                ? props.description
+                                : `Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia,
```

Comment:
Let the parent pass the Lorem Ispum. If you want to show some content here use "No content found" or a similar label.

Why is the this call the "NftModel"? Looks like a generic modal with some content.